crates/nassun/src/fetch/npm.rs

```diff
@@ -145,3 +145,150 @@ impl PackageFetcher for NpmFetcher {
         Ok(self.client.stream_external(url).await?)
     }
 }
+
+#[cfg(test)]
+mod test {
+    use node_semver::{Range, Version};
+    use oro_package_spec::VersionSpec;
+    use tempfile::tempdir;
+
+    use super::*;
+
+    #[tokio::test]
+    async fn read_name() -> Result<()> {
+        let fetcher = NpmFetcher {
+            client: oro_client::OroClient::default(),
+            registries: HashMap::default(),
+            corgi_packuments: DashMap::default(),
+            packuments: DashMap::default(),
+        };
+        let spec = PackageSpec::Npm {
+            scope: None,
+            name: "npm".to_string(),
+            requested: Some(VersionSpec::Range(Range::parse(">1.0.0").unwrap())),
```

Comment:
`">1.0.0".parse()?` should do the trick here :)

Reply:
Yup.